portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>Roland's - Portfolio</title>
+<link rel="stylesheet" href="style.css">
+<script src="script.js"></script>
+
+<nav id="navbar">
+	<div id="nav-left-side">
+		<a href="/index.html" class="nav-button navbar-item">NR</a>
+		<a href="/index.html" class="nav-button navbar-item">Home</a>
+		<a href="/works.html" class="nav-button navbar-item">Work</a>
+		<a href="/contact.html" class="nav-button navbar-item">Contact</a>
+	</div>
+	<div id="nav-right-side">
+		<a href="https://www.linkedin.com/in/roland-naijuka" target="_blank" class="nav-button navbar-item">LinkedIn</a>
+		<a href="mailto:naijuka.roland@outlook.com" target="_blank" class="nav-button navbar-item">Email</a>
+	</div>
+</nav>
+
+<div id="contact-me">
+	<h1>Contact Me!</h1>
+	<form action="#" method="POST">
+        <div class="form-content">
+            <label for="name">Name</label>
+            <input type="text" id="name" aria-describedby="userName"><br>
```

Comment:
aria-describedBy takes in the ID of an element whose text/value you want to be read by screenreaders.  You don't have an element with id="userName", so this doesn't do anything here.

If you want to provide a screenreader value, you can use `aria-label="My screenreader value here..."` instead.  Note, aria tags are pretty complex only work on certain elements, so I'm not sure they even work on inputs

Reply:
I googled and I don't need to add aria tags because there is already visible text